### PR TITLE
Added DaCy to universe

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
### Types of change
Documentation/universe update

## Checklist

- [x] I have submitted the spaCy Contributor Agreement.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/meta/universe.json [hunk 1]
```diff
@@ -888,6 +888,31 @@
...
+        {
+            "id": "dacy",
+            "title": "DaCy",
+            "slogan": "An efficient Pipeline for Danish NLP",
+            "description": "DaCy is a Danish preprocessing pipeline trained in SpaCy. It has achieved State-of-the-Art performance on Named entity recognition, part-of-speech tagging and dependency parsing for Danish. This repository contains material for using the DaCy, reproducing the results and guides on usage of the package. Furthermore, it also contains a series of behavioural test for biases and robustness of Danish NLP pipelines.",
+            "github": "centre-for-humanities-computing/DaCy",
+            "pip": "dacy",
+            "code_example": [
+                "import dacy",
+                "dacy.models() # get a list of dacy models",
```
Comment (adrianeboyd) [inline website/meta/universe.json:900]: We're happy to add this! I noticed that this line doesn't show anything in the demo script?

Comment (KennethEnevoldsen) [reply website/meta/universe.json:900]: Glad to hear! - I am not entirely sure I understand the last part though @adrianeboyd?

Comment (ines) [reply website/meta/universe.json:900]: I think what @adrianeboyd meant is that `dacy.models()` returns the list of models so just running the example script as is wouldn't output anything, which is potentially confusing. So in that case, maybe one of the following would be clearer:

```suggestion
                "print(dacy.models()) # get a list of dacy models",
```

```suggestion
                "model_names = dacy.models() # get a list of dacy models",
```

Comment (KennethEnevoldsen) [reply website/meta/universe.json:900]: Ahh thanks for the clarification - I have changed it to print now

**Pull request merged**